pkg/reconcile/pipeline/infinispan/handler/provision/gossiprouter.go

```diff
@@ -129,33 +129,43 @@ func GossipRouter(i *ispnv1.Infinispan, ctx pipeline.Context) {
 		}
 
 		router.Labels = routerLabels
+
+		container := &corev1.Container{
+			Name:           GossipRouterContainer,
+			Image:          i.ImageName(),
+			Command:        []string{"/opt/gossiprouter/bin/launch.sh"},
+			Args:           args,
+			Env:            []corev1.EnvVar{{Name: "ROUTER_JAVA_OPTIONS", Value: i.Spec.Container.RouterExtraJvmOpts}},
+			Ports:          containerPorts,
+			LivenessProbe:  TcpProbe(probePort, 10, 5, 0, 1, 60),
+			ReadinessProbe: TcpProbe(probePort, 10, 5, 0, 1, 60),
```

Comment:
The default value for `Probe.PeriodSeconds` is 10 seconds if the configured value is < 1. This seems high for the readiness probe, should it be smaller?